csep/utils/plots.py

```diff
@@ -1232,29 +1232,84 @@ def _get_marker_style(obs_stat, p, one_sided_lower):
             fmt = 'gs'
     return fmt
 
-def plot_comparison_test(results, plot_args=None):
-    """Plots list of T-Test or W-Test Results"""
+def _get_marker_t_color(distribution):
+    """Returns matplotlib marker style as fmt string"""
+    if distribution[0] > 0. and distribution[1] > 0.:
+        fmt = 'green'
+    elif distribution[0] < 0. and distribution[1] < 0.:
+        fmt = 'red'
+    else:
+        fmt = 'grey'
+
+    return fmt
+
+def _get_marker_w_color(distribution):
+    """Returns matplotlib marker style as fmt string"""
+
+    if distribution < 0.05:
```

Comment:
should we also add the percentile argument for this plot as well?

Reply:
done